src/odb/README.md

```diff
@@ -51,24 +51,12 @@ The full set of the Tcl commands exposed can be found under
 
 ### Python
 
-After building successfully, run a Python shell using `python3`. Load
-`opendbpy` module using:
-
-```
-import importlib.util
-spec = importlib.util.spec_from_file_location("opendbpy", "./build/src/swig/python/opendbpy.py")
-odb = importlib.util.module_from_spec(spec)
-spec.loader.exec_module(odb)
-
-# use it as following
-odb.[class_name]
-```
-
-You can find examples on using the API from Python under `test/python/`
-directory.
+After building successfully, run `openroad -python` to enable the Python
+interpreter. You can find examples on using the API from Python under
+`test/python/` directory.
 
 The full set of the Python classes exposed can be found under
-`./build/src/swig/python/opendbpy.py`.
+`./build/src/openroad_swig_py.py`.
```

Comment:
That doesn't really show all the classes - the db classes aren't in there.  I suggest in "openroad -python" do:

import openroad
import odb
dir(openroad)
dir(odb)

Reply:
possible fix with 4b5f84d